README.md

```diff
@@ -74,23 +75,27 @@ You will need a laptop with a Mac, Linux, or Windows operating system (not a tab
 
 **On your machine**
 - the Bash shell; you should only have to set this up if you are using a Windows machine, following the [Software Carpentry setup instructions](http://carpentries.github.io/workshop-template/#setup)
-- Git [installed](http://carpentries.github.io/workshop-template/#setup) and [configured](https://swcarpentry.github.io/git-novice/02-setup.html) on your local machine
-- Python 3.x; we recommend installing [Miniconda](https://docs.anaconda.com/miniconda/)
-- a text editor, preferably one designed for writing code; we recommend [VS Code](https://code.visualstudio.com) if you haven't chosen one.
+- Git [installed](https://carpentries.github.io/workshop-template/#git-1) and [configured](https://swcarpentry.github.io/git-novice/02-setup.html) on your local machine to attribute commits to you and your email address. Note for windows users: if you do not already have git set up locally, you can choose either git bash or installing the windows subsystem for linux. Both install options are included in the link above.
```

Comment:
What about this then?
```suggestion
- Git [installed and configured](https://swcarpentry.github.io/git-novice/) on your local machine to attribute commits to you and your email address. Note for windows users: if you do not already have git set up locally, you can choose either git bash or installing the windows subsystem for linux. Both install options are included in the link above.
```

Reply:
yeap, the new one is all good, the one above was just a half thought through workaround for Tim's comment